shell/src/main/java/alluxio/cli/fsadmin/journal/QuorumElectCommand.java

```diff
@@ -69,6 +71,15 @@ public int run(CommandLine cl) throws IOException {
       final int TIMEOUT_3MIN = 3 * 60 * 1000; // in milliseconds
       CommonUtils.waitFor("Waiting for election to finalize", () -> {
         try {
+          GetTransferLeaderMessagePResponse transferMsg = jmClient.getTransferLeaderMessage();
+          Optional<TransferLeaderMessage>
+                  exceptionMsgOpt = transferMsg.getTransMsgList().stream()
+                  .filter(TransferLeaderMessage::getIsException).findAny();
+          String msg = exceptionMsgOpt.isPresent() ? exceptionMsgOpt.get().getMsg() : null;
+          if (msg != null) {
+            System.out.println(msg);
+            return false;
```

Comment:
If you return false here, the `waitFor` loop will continue until 30 minutes past, so would you like to throw an exception to terminate the `waitFor` loop? So that the it can be found at once.